### ISSUE TITLE
ExID is ptUnknown where it should be ptLibrary.

### ISSUE THREAD

**Issue by JBontes** [RomanYankovsky/DelphiAST#167]
After issue #164 is fixed, deprecated and platform process correctly, but somehow `library` is processed differently.

Demo:

```delphi
unit LibraryOnConst;

interface

const
  MyConst = 'test' deprecated 'Do not use';  //works
  MyConst2 = 'test2' platform;  //works
  MyConst3 = 'test4' library; //does not work. ExID does not get processed correctly.

implementation

end.
```

**Comment by RomanYankovsky** [comment]
You have to check TokenID in order to get ptLibrary value for that token, not ExID. 

ExID is for special cases. In Delphi some keywords cannot be used as identifiers (like 'Library' or 'Unit') but some keywords can also be used as identifiers (like 'Helper' or 'Deprecated'). In the first case you may check TokenID value and that's enough. In the seconds case TokenID=ptIdentifier and you have to check ExID value.

Sounds complicated, I know :)

**Comment by RomanYankovsky** [comment]
So there was code that expected to see ptLibrary value in ExID and that was a bug. Fixed by 224ed8094da8e113715bbb2720a31c2876101a15
